egs/wsj/s5/utils/text_to_phones.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Johns Hopkins University (author: Hossein Hadian)
+# Apache 2.0
+
+
+""" This reads data/train/text from std input and converts the word transcriptions
+    to phone transcriptions using the provided lexicon.
+"""
+
+import argparse
+import os
+import sys
+import copy
+import math
+import random
+
+parser = argparse.ArgumentParser(description="""This script reads
```

Comment:
This can also be put into functions, but since this is a small script it may not be essential.